platform/lib/platform.js

```diff
@@ -64,6 +65,9 @@ class Platform {
           log.success(`server listening on ${PORT}!`);
           resolve();
         });
+
+        webSocketServer.startLiveBlog(this.httpServer);
```

Comment:
This is our only socket server, so I wouldn't make the start name specific to the liveblog sample.

Reply:
Ah, I was planning for future socket servers. I can change this if you like. (As apparently you do)

Reply:
Done